src/site/content/sw-payload.njk

```diff
@@ -2,8 +2,7 @@
 lang: ''
 layout: layout
 nometa: true
-permalink: /sw-partial-layout.partial
-show_banner: false
+permalink: /sw-payload
```

Comment:
I think you need to add `show_banner: false` back, otherwise it displays twice on any page with a banner (see the percy tests)